index.rst

```diff
@@ -122,6 +124,29 @@ Indices
 - :ref:`genindex`
 - :ref:`Search <search>`
 
+.. _part-usage:
+
+Use of Rubin Observatory Software
+=================================
```

Comment:
```suggestion
.. _part-citation:

Citing and acknowledging the LSST Science Pipelines
===================================================
```

I think "Use of Rubin Observatory Software" doesn't hit the mark for a couple reasons:

1. People are looking for information on how to acknowledge and/or cite the LSST Science Pipelines in their works. I think it's smart to key the header on that use case. Further, I'd recommend splitting the licensing information into another section, below this (see comment below).
2. "Rubin Observatory Software" is a fairly broad term, and this section doesn't accurately apply to Rubin software in general, only the LSST Science Pipelines component of it. So I think it's good to use "LSST Science Pipelines" here.
3. On a style basis, note this header is in title case, whereas [our style](https://developer.lsst.io/user-docs/index.html) specifies sentence case for section headings.